src/core/lib/security/security_connector/security_connector.h

```diff
@@ -238,6 +238,9 @@ grpc_security_status grpc_ssl_server_security_connector_create(
 const tsi_peer_property* tsi_peer_get_property_by_name(const tsi_peer* peer,
                                                        const char* name);
 
+/* Hostname verification entry point. */
+int ssl_host_matches_name(const tsi_peer* peer, const char* peer_name);
```

Comment:
Could you please move this declaration to the section of "exposed for testing only" section?